README.md

```diff
@@ -160,38 +111,23 @@ Here are some of the planned improvements:
 - [x] [Windowed aggregations over Tumbling & Hopping windows](https://quix.io/docs/quix-streams/v2-0-latest/windowing.html)
 - [x] [State recovery based on Kafka changelog topics](https://quix.io/docs/quix-streams/advanced/stateful-processing.html#fault-tolerance-recovery)
 - [x] [Group-by operation](https://quix.io/docs/quix-streams/groupby.html)
-- [X] ["Exactly Once" delivery guarantees for Kafka message processing (AKA transactions)](https://quix.io/docs/quix-streams/configuration.html#processing-guarantees)
+- [x] ["Exactly Once" delivery guarantees for Kafka message processing (AKA transactions)](https://quix.io/docs/quix-streams/configuration.html#processing-guarantees)
 - [ ] Joins
 - [ ] Windowed aggregations over Sliding windows
 - [ ] Support for Avro and Protobuf formats
 - [ ] Schema Registry support
 
 
-To find out when the next version is ready, make sure you watch this repo 
-and join our [Quix Community on Slack](https://quix.io/slack-invite)! 
+## Get Involved 👋
+##//TODO: Make the items shorter.
 
-## Contribution Guide
+- Please use [GitHub issues](https://github.com/quixio/quix-streams/issues) to report bugs and suggest new features.
+- Follow us on [Twitter](https://twitter.com/quix_io) and [Linkedin](https://www.linkedin.com/company/70925173) where we share our latest tutorials, forthcoming community events and the occasional meme.
+- Join the [Quix Community on Slack](https://quix.io/slack-invite), a vibrant group of Python developers, data enthusiasts and newcomers to Apache Kafka, who are learning and leveraging Quix Streams for real-time data processing.
+- If you have any questions or feedback - write to us at support@quix.io!
```

Comment:
I would also add:
- Watch and subscribe to [@QuixStreams on YouTube](https://www.youtube.com/@QuixStreams) for code-along tutorials from scratch and interesting community highlights.

Reply:
I'd also adjust the order of items by priority:
- GitHub issues
- Slack
- YouTube
- X and LinkedIn
- questions/feedback